mesheryctl/internal/cli/root/system/stop.go

```diff
@@ -123,25 +127,51 @@ func stop() error {
 			}
 		}
 
-		log.Info("Stopping Meshery resources...")
+		cli, err := client.NewClientWithOpts(client.FromEnv, client.WithAPIVersionNegotiation())
+		if err != nil {
+			return errors.Wrap(err, "unable to create docker client")
+		}
+
+		spinner := utils.CreateDefaultSpinner("Removing meshery containers in docker\n", "")
+		spinner.Start()
 
-		// Stop all Docker containers
-		stop := exec.Command("docker-compose", "-f", utils.DockerComposeFile, "stop")
-		stop.Stdout = os.Stdout
-		stop.Stderr = os.Stderr
+		options := types.ContainerListOptions{
+			Filters: filters.NewArgs(
+				filters.Arg("name", "meshery"),
+				filters.Arg("network", "meshery_default"),
+				filters.Arg("label", "com.centurylinklabs.watchtower.enable=true"),
+			),
+		}
+		contaienrs, err := cli.ContainerList(context.Background(), options)
```

Comment:
handle error calls

Reply:
@Philip-21 do I create new error codes in [system error file](https://github.com/meshery/meshery/blob/master/mesheryctl/internal/cli/root/system/error.go)?